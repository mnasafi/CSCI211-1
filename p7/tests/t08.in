echo t08: Insert days of week into tree, then call find on several strings NOT in tree.
insert Saturday
insert Friday
insert Tuesday
insert Monday
insert Thursday
insert Wednesday
insert Sunday
echo Calling find on <Mon> (should NOT be in tree):
find Mon
echo Calling find on <Sun> (should NOT be in tree):
find Sun
echo Calling find on <December> (should NOT be in tree):
find December
echo Calling find on <a> (should NOT be in tree):
find a
