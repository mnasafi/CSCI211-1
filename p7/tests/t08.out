t08: Insert days of week into tree, then call find on several strings NOT in tree.
Calling find on <Mon> (should NOT be in tree):
<Mon> is not in tree.
Calling find on <Sun> (should NOT be in tree):
<Sun> is not in tree.
Calling find on <December> (should NOT be in tree):
<December> is not in tree.
Calling find on <a> (should NOT be in tree):
<a> is not in tree.
